### PR TITLE
Smooch Connect docs

### PR DESCRIPTION
**Pull request by alavers**
@jugarrit @mspensieri @dannytranlx @lemieux

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+                        'rm -rf build',
+                        'bundle exec middleman build --clean'
+                    ].join(' && ');
+                }
+            },
+            checkoutDocs: {
+                cmd: function() {
+                    return [
+                        'rm -rf ' + githubPagesDir,
+                        'git clone git@github.com:alavers/smooch-docs.git --branch gh-pages --single-branch ' + githubPagesDir,
```
Comment (mspensieri) [inline Gruntfile.js]: leftover change?

File: source/includes/rest/_core.md [hunk 2]
```diff
@@ -1920,3 +1920,101 @@ smooch.menu.remove()
...
+
+| *Error Code* | *Delivery Method* | *Description* |
+|--------------|-------------------|---------------|
+| **access_denied** | 302 | The user denied access |
+
+## Token Endpoint
+
+> Request:
+
+```
```
Comment (mspensieri) [inline source/includes/rest/_core.md]: Should probably follow [this convention](https://github.com/smooch/smooch-docs/pull/123/files#diff-75d028b5bdf33b16b22025eff42af949R62) for the javascript tab

File: source/includes/rest/_core.md [hunk 3]
```diff
@@ -1920,3 +1920,101 @@ smooch.menu.remove()
...
 
 Remove the specified app's menu.
+
+# Smooch Connect
+
+Smooch has a rapidly growing [partner ecosystem](https://smooch.io/partners/). If you have a product or service that could benefit from Smooch's connected channels, messaging features, and more, we'd love to offer it on the Smooch marketplace. Smooch Connect makes this possible. Smooch uses [OAuth](https://tools.ietf.org/html/rfc6749) to facilitate the installation of parter integrations.
+
+First, you should [contact us](https://smooch.io/partners/) about becoming a partner and we'll provision you with a client id and everything else you'll need to build your end of the OAuth flow. In the meantime the OAuth endpoints detailed below can be tested by impersonating Shoplifter, a partnership we built for testing purposes.
+
+## Add to Smooch Button
```
Comment (jpjoyal) [inline source/includes/rest/_core.md]: @kvgauthier did you think about the rest of the content you think would fit here ?

File: source/includes/rest/_core.md [hunk 4]
```diff
@@ -1920,3 +1920,101 @@ smooch.menu.remove()
 <api>`DELETE /v1/menu`</api>
 
 Remove the specified app's menu.
+
+# Smooch Connect
+
+Smooch has a rapidly growing [partner ecosystem](https://smooch.io/partners/). If you have a product or service that could benefit from Smooch's connected channels, messaging features, and more, we'd love to offer it on the Smooch marketplace. Smooch Connect makes this possible. Smooch uses [OAuth](https://tools.ietf.org/html/rfc6749) to facilitate the installation of parter integrations.
+
+First, you should [contact us](https://smooch.io/partners/) about becoming a partner and we'll provision you with a client id and everything else you'll need to build your end of the OAuth flow. In the meantime the OAuth endpoints detailed below can be tested by impersonating Shoplifter, a partnership we built for testing purposes.
```
Comment (jpjoyal) [inline source/includes/rest/_core.md]: I think its worth to mention up front what URLs they'll need to provide us just to remove some back and forth down the road.

Comment (jpjoyal) [inline source/includes/rest/_core.md]: @alavers do you think a section with our sequence chart would make sense here ?

File: Gruntfile.js (hunk 1 above)
Comment (alavers) [inline Gruntfile.js]: Oh yeah... :)

File: source/includes/rest/_core.md (hunk 4 above)
Comment (alavers) [inline source/includes/rest/_core.md]: I don't see why not. @kvgauthier ?

File: source/includes/rest/_core.md [hunk 5]
```diff
@@ -1920,3 +1920,101 @@ smooch.menu.remove()
 <api>`DELETE /v1/menu`</api>
 
 Remove the specified app's menu.
+
+# Smooch Connect
+
+Smooch has a rapidly growing [partner ecosystem](https://smooch.io/partners/). If you have a product or service that could benefit from Smooch's connected channels, messaging features, and more, we'd love to offer it on the Smooch marketplace. Smooch Connect makes this possible. Smooch uses [OAuth](https://tools.ietf.org/html/rfc6749) to facilitate the installation of parter integrations.
```
Comment (kvgauthier) [inline source/includes/rest/_core.md]: Smooch has a rapidly growing partner ecosystem which helps you unlock messaging in your product or service and let customers find you alongside complementary partners. If you're interesting in becoming a partner, tell us what you're building with Smooch using our partner application and we'll provision you with a client id and everything else you'll need to build your end of the OAuth flow using Smooch Connect. In the meantime the OAuth endpoints detailed below can be tested by impersonating Shoplifter, a partnership we built for testing purposes.

File: source/includes/rest/_core.md (hunk 4 above)
Comment (kvgauthier) [inline source/includes/rest/_core.md]: What is the sequence chart?

File: source/includes/rest/_core.md (hunk 3 above)
Comment (kvgauthier) [inline source/includes/rest/_core.md]: not really, How can I get a clean preview of this new doc?

File: source/includes/rest/_core.md (hunk 4 above)
Comment (jpjoyal) [inline source/includes/rest/_core.md]: this

File: source/includes/rest/_core.md (hunk 3 above)
Comment (jpjoyal) [inline source/includes/rest/_core.md]: @alavers I've got kevin setup with the docs repo on his machine. Just need to get him to checkout your remote branch

@kvgauthier see http://stackoverflow.com/questions/5884784/how-to-pull-remote-branch-from-somebody-elses-repo

**Pull request merged**